packages/google-pubsub-microservice/src/GooglePubSubServer.ts

```diff
@@ -23,7 +23,9 @@ export class GCPubSubServer extends Server implements CustomTransportStrategy {
    */
   protected readonly logger: Logger = new Logger(GCPubSubServer.name);
 
-  constructor(private readonly options?: GooglePubSubOptions & { listenOptions?: GCListenOptions }) {
+  constructor(
+    private readonly options?: GooglePubSubOptions & { listenOptions?: GCListenOptions } & { topicsNames?: string[] },
```

Comment:
You can merge these options

```suggestion
    private readonly options?: GooglePubSubOptions & { listenOptions?: GCListenOptions; topicsNames?: string[] },
```